projects/owi/programs/owi/module.nix

```diff
@@ -0,0 +1,21 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+let
+  cfg = config.programs.owi;
+in
+{
+  options.programs.owi = {
```

Comment:
Side note to @eljamm: this pattern is repeating itself so often, we could actually try applying the portable services idea to it. I.e. make (the equivalent of) `programs` an `attrsOf (submoduleWith { modules = [ program ]; })` and set instances as an attr with only the package specified in the simplest case. But that allows mixing in additional aspects or changing (in a central place!) what a program is made of and what API it exposes. And for consumers it allows having multiple instances of a program in a system and addressable by name, even referring to each other in their customization options.

Reply:
I'm not sure. Let's discuss this in more details later and we can perhaps create a User Story for it.